pkg/api/api.go

```diff
@@ -192,6 +195,7 @@ func (a *StreamplaceAPI) Handler(ctx context.Context) (http.Handler, error) {
 	addHandle(apiRouter, "GET", "/api/bluesky/resolve/:handle", a.HandleBlueskyResolve(ctx))
 	addHandle(apiRouter, "GET", "/api/view-count/:user", a.HandleViewCount(ctx))
 	addHandle(apiRouter, "GET", "/api/clip/:user/:file", a.HandleClip(ctx))
+	addHandle(apiRouter, "GET", "/api/og/:username/og.jpg", a.HandleOGImage(ctx))
```

Comment:
Let's ship this as a new XRPC so it's documented and stuff. Perhaps `place.stream.live.getTitleCard` or something like that? Let me know if you get in trouble with the codegen